pkg/kubelet/server/stats/summary_sys_containers_windows.go

```diff
@@ -28,7 +28,9 @@ import (
 )
 
 func (sp *summaryProviderImpl) GetSystemContainersStats(nodeConfig cm.NodeConfig, podStats []statsapi.PodStats, updateStats bool) (stats []statsapi.ContainerStats) {
-	stats = append(stats, sp.getSystemPodsCPUAndMemoryStats(nodeConfig, podStats, updateStats))
+	containerStats := sp.getSystemPodsCPUAndMemoryStats(nodeConfig, podStats, updateStats)
+	sp.addSystemPodsSwapStats(podStats, &containerStats)
```

Comment:
Would it be better if we call `sp.addSystemPodsSwapStats(podStats, &containerStats)` inside `sp.getSystemPodsCPUAndMemoryStats(nodeConfig, podStats, updateStats)` so we don't have to retrieve and append swap stats here?

Reply:
If we would do that then swap statistics would also exist for the second method call (`func (sp *summaryProviderImpl) GetSystemContainersCPUAndMemoryStats()`). Since swap is considered as a different resource than memory and CPU, and since in other methods I've included swap stats only for `GetAllStats()` methods and not for `GetMemoryAndCpuStats()` methods I think it would be more consistent to keep it this way.